cico_setup.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -ex
+
+load_jenkins_vars() {
+    if [ -e "jenkins-env" ]; then
+        cat jenkins-env \
+          | grep -E "(DEVSHIFT_TAG_LEN|DEVSHIFT_USERNAME|DEVSHIFT_PASSWORD|JENKINS_URL|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId)=" \
```

Comment:
copr token will have to be grepped here as well. @msrb correct me if I am wrong.